src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -163,35 +168,49 @@ PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProperties, U
                     field.setValue(getYesOrNoValue(dar.getBoolean(DarConstants.PEDIATRIC)));
                     break;
                 }
-                case DarConstants.ILLEGAL_BEHAVE: {
+                case DarConstants.ILLEGAL_BEHAVE:
+                    // Handle legacy "onegender"
```

Comment:
Need to adjust the comment for illegalBehavior

Reply:
Yeah, I'm reviewing myself and just caught a couple of these!